docs/database/db_schema-main.sql

```diff
@@ -59,6 +58,20 @@ CREATE TABLE `certificateStatus` (
   PRIMARY KEY (`serial`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+CREATE TABLE `challenges` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `authorizationID` varchar(255) DEFAULT NULL,
+  `type` varchar(255) NOT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  `error` varchar(255) DEFAULT NULL,
+  `validated` datetime DEFAULT NULL,
+  `uri` varchar(255) DEFAULT NULL,
+  `token` varchar(255) NOT NULL,
+  `tls` tinyint(1) DEFAULT NULL,
+  `validation` varchar(1536) DEFAULT NULL,
```

Comment:
Example of a concern I have about keeping the json and db models the same: the validation json seems like they should be blobs.

Similarly, I think we're silently truncating uri here and other things (maybe error? or token?)